mmdeploy/codebase/mmaction/deploy/video_recognition_model.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+from typing import Any, List, Optional, Sequence, Union
+
+import mmengine
+import torch
+from mmaction.utils import LabelList
+from mmengine import Config
+from mmengine.registry import Registry
+from mmengine.structures import BaseDataElement, LabelData
+
+from mmdeploy.codebase.base import BaseBackendModel
+from mmdeploy.utils import (Backend, get_backend, get_codebase_config,
+                            get_root_logger, load_config)
+
+__BACKEND_MODEL = Registry('backend_video_recognizer')
+
+
+@__BACKEND_MODEL.register_module('end2end')
+class End2EndModel(BaseBackendModel):
+    """End to end model for inference of video recognition.
+
+    Args:
+        backend (Backend): The backend enum, specifying backend type.
+        backend_files (Sequence[str]): Paths to all required backend files(e.g.
+            '.onnx' for ONNX Runtime, '.param' and '.bin' for ncnn).
+        device (str): A string represents device type.
+        deploy_cfg (str | mmengine.Config): Deployment config file or loaded
+            Config object.
+        model_cfg (str | mmengine.Config): Model config file or loaded Config
+            object.
+    """
+
+    def __init__(self,
+                 backend: Backend,
+                 backend_files: Sequence[str],
+                 device: str,
+                 deploy_cfg: Union[str, Config] = None,
+                 model_cfg: Union[str, Config] = None,
+                 **kwargs):
+        super(End2EndModel, self).__init__(deploy_cfg=deploy_cfg)
+        model_cfg, deploy_cfg = load_config(model_cfg, deploy_cfg)
+        from mmaction.registry import MODELS
+        self.data_preprocessor = MODELS.build(
```

Comment:
It seems that some model in mmaction2 does not have a data_preprocessor like https://github.com/open-mmlab/mmaction2/blob/dev-1.x/configs/_base_/models/bmn_400x100.py
Should we check it?